src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,29 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    protected LocalDate by;
+    String DATE_FORMAT = "MMM d yyyy";
+    String DIVIDER = " | ";
```

Comment:
Should these variables be static final? What about the access modifier?

... I noticed the same issue in several other places too